### PR TITLE
New environment variables

### PR DESCRIPTION
**Pull request by DasCanard**
New environment variables:
- Prefix
- Discord_ClientID
- Discord_ClientSecret
- Website
- Spotify_ClientID
- Spotify_ClientSecret

Also added in app.json

### CODE REVIEW

**Comment by DasCanard** [comment]
> Have you tested this to see if this works?

I testet the environment variables but not the heroku deploy but it should work.
I will test it now

**Comment by DasCanard** [comment]
Its working fine

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Got any error or any problem with the code? Join this server and ask your questi
 
 Glitch: [![Remix on Glitch](https://cdn.glitch.com/2703baf2-b643-4da7-ab91-7ee2a2d00b5b%2Fremix-button.svg)](https://glitch.com/edit/#!/import/github/SudhanPlayz/Discord-MusicBot)
 Repl: [![Run on Repl.it](https://repl.it/badge/github/SudhanPlayz/Discord-MusicBot)](https://repl.it/github/SudhanPlayz/Discord-MusicBot)
-Heroku: [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/SudhanPlayz/Discord-MusicBot)
+Heroku: [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/DasCanard/Discord-MusicBot/)
```
[the remark this answers is not in the document]
Comment (DasCanard) [reply README.md:22]: For testing...
I thought I was in a different branch

[the remark this answers is not in the document]
Comment (joeyk710) [reply README.md:22]: can you change it back? we won't accept that since it directs it under your name

**Comment by joeyk710** [comment]
alright so you tested this and it works?

**Pull request merged**